src/state/pstake/hooks/common.ts

```diff
@@ -30,39 +30,62 @@ export const useMinichefPools = (): { [key: string]: number } => {
   }, [lpTokensArr]);
 };
 
-export function useMinichefPendingRewards(miniChefStaking: DoubleSideStakingInfo | null) {
+/**
+ * This hook return the pending rewards and multipliers of extra tokens give by farm
+ * @param miniChefStaking staking info with farm values
+ * @returns rewardTokensAmount is an array of tokens pending rewards,
+ * rewardTokensMultiplier is an array of tokens multipliers
+ */
+export function useExtraPendingRewards(miniChefStaking: DoubleSideStakingInfo | null) {
```

Comment:
what is the reason behind changing name of this hook ? because looks like this is specifically for minichef right ?

Reply:
Because before we only used it for mini chef, a few months ago we started using it for pangochef too and that made the name outdated